ftw/collectionblock/__init__.py

```diff
@@ -2,3 +2,9 @@
 
 
 _ = MessageFactory('ftw.collectionblock')
+
+
+def initialize(context):
+    """Required for being able to uninstall this product.
+    """
+    pass
```

Comment:
Not necessary - the doc string itself is enough

Reply:
👍